recipes/liboptv/meta.yaml

```diff
@@ -14,12 +14,18 @@ requirements:
     - {{ compiler('cxx') }}
     - python
     - setuptools
-    - numpy=1.16
+    - numpy x.x
     - cython
     - pyyaml
```

Comment:
I'd check out the docs here for how to build with numpy. https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy